python/lsst/ip/diffim/detail/__init__.py

```diff
@@ -19,4 +19,4 @@
 # the GNU General Public License along with this program.  If not,
 # see <https://www.lsstcorp.org/LegalNotices/>.
 #
-from .diffimDetailLib import *
+from .._ipdiffimLib import *
```

Comment:
I hate the fact that we're exposing things in the `detail` namespace to Python at all, but it's preexisting (and probably so tests could be written in Python), so I suppose this isn't the right ticket to do anything about that.